app/services/code_climate/projects_without_c_c.rb

```diff
@@ -0,0 +1,43 @@
+module CodeClimate
+  class ProjectsWithoutCC < BaseService
```

Comment:
I'd like this service to be properly tested in its own spec file covering edge cases

Reply:
Will address on next PR.

Reply:
Will address on next PR.